.github/workflows/acceptance-tests.yml

```diff
@@ -5,38 +5,41 @@ on:
   schedule:
     - cron: "0 4 * * *" # workflow runs every day at 04:00 AM
   workflow_call: # workflow runs after code-health
-    inputs:
-      parent-event-name:
-        required: true
-        type: string
   pull_request: # you can run a specic job in your PR using GitHub labels
     types: [ labeled ]
 
+env:
+  ALWAYS_RUN: ${{ github.event_name == 'workflow_dispatch' || github.event_name == 'schedule' || github.event.label.name == 'run-testacc' }}
+  CI: true
+  PARALLEL_GO_TEST: 20
+  ACCTEST_TIMEOUT: ${{ vars.ACCTEST_TIMEOUT }}
+  SKIP_TEST_EXTERNAL_CREDENTIALS: ${{ vars.SKIP_TEST_EXTERNAL_CREDENTIALS }}
+  TF_LOG: ${{ vars.LOG_LEVEL }}
```

Comment:
I left this comment in the other PR but I am sorry I missed your eventual response. I have a preference in not having global env variables because it goes in the opposite direction of what you want to do here, which IIUC is simplify and "isolate" the context of individual jobs.

I suggest we leave these at each job level, they will be repeated, yes, but verbosity is not necessarily bad

Reply:
the other PR is stil open. Basically my argument is that it's ok to have common variables for things that are common and we don't want to change in each test. 
As an example we want to say CI: true for all tests run from here.   I don't see a benefit in having in each single tests, and there is the problem of forgetting it in some tests as you copy/paste.

Reply:
I do see the value in extracting the common env variables, these will apply to all acceptance test jobs and remove noise from more specific configurations each one has. This can also help when adding new common env variables such as the terraform version used to run acceptance tests.
On the other hand having the criteria `ALWAYS_RUN` here is a little confusing, having it inlined should be okay.

Reply:
@AgustinBettati very good point about TF version variables

Reply:
this reminds me a lot of the best practices for testing.
I remember in Google there was a best practice where "variables, methods, context etc.etc" should be as close as possible (preferably inside) the test itself, so the reader of that (functional/unit/integ) test does not have to move in the file to understand what's going on. As a consequence you would see lots of duplication in the code because same variables, same init logic would be repeated instead of being separated. But this was preferred and recommended. And to me it made lots of sense.

This reminds me of that, we are trying to make this config file more organised at the price of "intuitiveness" and "time-to-understand". 

Said that, I won't push back too much on this - if you both like this approach then I am fine! :)

Reply:
and we're removing 100 lines ;-)

i see your point, let's discuss offline